cli/src/pcluster/templates/cdk_builder_utils.py

```diff
@@ -742,6 +742,17 @@ def _build_policy(self) -> List[iam.PolicyStatement]:
                             )
                         ],
                     ),
+                    iam.PolicyStatement(
+                        sid="ElasticLoadBalancingDescribe",
+                        actions=[
+                            "elasticloadbalancing:DescribeLoadBalancers",
+                            "elasticloadbalancing:DescribeTags",
+                            "elasticloadbalancing:DescribeTargetGroups",
+                            "elasticloadbalancing:DescribeTargetHealth",
+                        ],
+                        effect=iam.Effect.ALLOW,
+                        resources=["*"],
+                    ),
```

Comment:
I would say this is the _wrong place_ for these policies. We should add these policies only if the custom munge key is used **and** login nodes are present.

Below in this very file there are a few examples of conditional policies for directory service and Slurm accounting (see line 774)..

Reply:
Makes sense. Thank you Jacopo for your reminding. Done! Now the policies will be added only if LoginNodes and Custom Munge Key are used.